src/display/process/allProcs.go

```diff
@@ -268,12 +279,44 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 							killSelected = false
 							myPage.ProcTable.CursorColor = selectedStyle
 						}
+						sendSignal = false
+						updateUI()
 					case "K", "<F9>":
 						// get process and kill it
```

Comment:
I'm a little confused, why retain this when we have the `<Enter>` and the numeric cases handled?

Reply:
Additionally, when this is done, the Kill Table is retained, it doesn't close.

Reply:
As I mentioned in the discussion about keybindings, this was retained as a convenience, since most of the time you want to kill the process. So double tapping K or F9 will do that without having to navigate the menu. However the menu should close after this, will fix that